### PR TITLE
Throw InacessibleContentError on empty matched content

### PR DESCRIPTION
**Pull request by Ndpnt**
Alternative to #751

### CODE REVIEW

**Review by martinratinaud — APPROVED** [review]
Nice thanks

File: src/archivist/filter/index.js [hunk 1]
```diff
@@ -80,7 +80,13 @@ export async function filterHTML({ content, documentDeclaration }) {
     }
   });
 
-  return transform(domFragment);
+  const markdownContent = transform(domFragment);
+
+  if (!markdownContent) {
+    throw new InaccessibleContentError(`The provided selector "${contentSelectors}" match an empty content in the web page at '${location}'`);
```
Comment (MattiSG) [inline src/archivist/filter/index.js:86]:
```suggestion
    throw new InaccessibleContentError(`The provided selector "${contentSelectors}" matches no content in the web page at '${location}'`);
```

**Review by MattiSG — CHANGES_REQUESTED** [review]
Can we please add another test with whitespace only? 🙂

**Comment by MattiSG** [comment]
Feel free to dismiss my “request changes” when my request is handled, I might not be reactive enough today 🙂
